### DISCUSSION TITLE
Authoring tag helpers

### DISCUSSION THREAD

**Discussion by nickshek10** [dotnet/aspnetcore#59501] [Q&A]
How do you use the syntax that the view component tag helper uses? I think it's quite neat and would like to try it. I can't find the source code anywhere and there doesn't appear to be any documentation.

<vc:**_viewcomponentname_** />

**Comment by gfoidl** [comment]
Are you looking for [Invoke a view component as a Tag Helper](https://learn.microsoft.com/en-us/aspnet/core/mvc/views/view-components?view=aspnetcore-9.0#invoke-a-view-component-as-a-tag-helper)?
